### ISSUE TITLE
Different contract address for Trades?

### ISSUE THREAD

**Issue by bt3new** [etherdelta/bots#10]
I'm wondering if it is possible to make trades on EtherDelta with another contract addresses than 0x8d12a197cb00d4747a1fe03395095ce2a5cc6819 ?

I noticed this account: https://etherscan.io/tx/0x77e904ddde677414d9c9307e57754783433040c4e957ea64bbc719d6b9050421

He is making trades with the contract address 0xf43b43869dd95e53923b30a79902915c1faa47d3

**Comment by zackcoburn** [comment]
Yes, it's possible to make a smart contract that trades with EtherDelta
(you still have to initiate the trade through a script). People use these
to lock in multiple trades at once.

**Comment by bt3new** [comment]
Good to know. Is there a template or reference for such smart contracts?

**Comment by zackcoburn** [comment]
Unfortunately, I've never seen anyone publish the smart contract code for
one of these bots. The people who make them are usually pretty competitive.
That being said, it wouldn't be a hard smart contract to write.

**Comment by bt3new** [comment]
Hm, okay. So I guess these smart contracts also improve the speed of tradings / transactions? Is it even possible to compete with bots using their own smart contracts when it comes down to transaction speed?

**Comment by zackcoburn** [comment]
You don't need to use a smart contract to compete with a bot, you just need
a bot yourself. The smart contract just gives you the ability to trade
multiple trades at once in such a way that you'll either trade all of the
trades or none of the trades (great for locking in an arbitrage).

Zack Coburn

NOTICE: This e-mail transmission (and/or the attachments accompanying it)
may contain confidential information belonging to the sender which is
protected by law. The information is intended only for the use of the
intended recipient. If you are not the intended recipient, you are hereby
notified that any disclosure, copying, distribution or the taking of any
action in reliance on the contents of this information is strictly
prohibited. Any unauthorized interception of this transmission is illegal.
If you have received this transmission in error, please promptly notify the
sender by reply e-mail, and then destroy all copies of the transmission.

**Comment by bt3new** [comment]
okay, thank you very much!

**Comment by JoeMalone** [comment]
Hi Zack

We are trying to issue a custom token on EtherDelta but I am having trouble with the order book,
I have sufficient balance of ether and the token. 

It just doesn't show up...It says that we can read order book with the transaction but we just need to wait...

What could be a solution to this problem?

Thanks 

Enjoy Chicago
